### ISSUE TITLE
claims are updating too often

### ISSUE THREAD

**Issue by tiger5226** [lbryio/chainquery#44]
These are logs from a local instance of lighthouse

```
2018-07-22T05:29:55.189Z - info: [Importer] Pushed 145998 claims to elastic search
2018-07-22T05:29:55.190Z - info: [Importer] Removing blocked claims from search!
2018-07-22T05:29:55.471Z - info: [Importer] Done processing blocked claims!
2018-07-22T05:42:13.262Z - info: [Importer] Pushed 190150 claims to elastic search
2018-07-22T05:42:13.263Z - info: [Importer] Removing blocked claims from search!
2018-07-22T05:42:13.740Z - info: [Importer] Done processing blocked claims!
2018-07-22T05:53:01.505Z - info: [Importer] Pushed 54861 claims to elastic search
2018-07-22T05:53:01.505Z - info: [Importer] Removing blocked claims from search!
2018-07-22T05:53:01.673Z - info: [Importer] Done processing blocked claims!
2018-07-22T06:04:19.952Z - info: [Importer] Pushed 112294 claims to elastic search
2018-07-22T06:04:19.952Z - info: [Importer] Removing blocked claims from search!
2018-07-22T06:04:20.118Z - info: [Importer] Done processing blocked claims!
```
As you can see, running ever 10 minutes there are 10s of thousands being updated in Chainquery. This is not real. We need to make sure if there are no changes we are not writing to the database. The only way this is close to true is if some of the most popular claim names have claims being updated, which wouldn't even be the `112,294` mentioned above.

**Comment by tiger5226** [comment]
I looked into this. The problem was that we were setting claims to active, saving, then setting them to controlling when we did that step. To resolve, I moved the setting of bid state to when we determine controlling status to avoid change claims to active only to change it back to controlling. 

I also fixed a few bugs I found:

**Comment by tiger5226** [comment]
solved with 66b40f04ce0f6b20887566b2f49c3bfd2453fc17

**Comment by tiger5226** [comment]
deployed with 0.5.4
